build.gradle

```diff
@@ -6,9 +6,10 @@ buildscript {
         maven {
             url "https://maven.google.com"
         }
+        google()
```

Comment:
Isn't this the same as the section above? Can we drop that section and move google up top to address the fdroid issue (on the forum)?

Reply:
Will try that.